packages/contracts/package.json

```diff
@@ -12,6 +12,8 @@
     "migrate:local": "hardhat run --network localhost scripts/deploy.ts",
     "migrate:kovan": "hardhat run --network kovan scripts/deploy.ts",
     "migrate:ropsten": "hardhat run --network ropsten scripts/deploy.ts",
+    "migrate:rinkeby": "hardhat run --network rinkeby scripts/deploy.ts",
+    "migrate:goerli": "hardhat run --network goerli scripts/deploy.ts",
```

Comment:
could see this becoming a single script that iterates over all networks in the future